render/hyperscript.js

```diff
@@ -5,7 +5,7 @@ var Vnode = require("../render/vnode")
 var selectorParser = /(?:(^|#|\.)([^#\.\[\]]+))|(\[(.+?)(?:\s*=\s*("|'|)((?:\\["'\]]|.)*?)\5)?\])/g
 var selectorCache = {}
 function hyperscript(selector) {
-	if (selector && typeof selector !== "string" && !selector.view) {
+	if (!selector || typeof selector !== "string" && !selector.view) {
 		throw Error("The selector should either be a string or an object with a view.");
```

Comment:
`selector == null` would be better because the empty string is falsy and equivalent to `'div'`. The loose equality with `null` also matches `undefined`.